docs/website/docs/guides/deployment-configurations/index.md

```diff
@@ -87,10 +87,11 @@ focus and the build configuration.
 | ------------ | ----------- |
 | `local-sync` | Synchronous local CPU device with inline execution |
 | `local-task` | Multithreaded local CPU device using a 'task' executor |
-| `vulkan`     | Portable GPU execution using the Vulkan API |
-| `hip`        | AMD GPU execution using HIP |
+| `amdgpu`     | **Experimental** <br> AMD GPU execution using HSA |
```

Comment:
Do you expect the AMDGPU driver built on HSA to have its own dedicated page on the website, or will it be merged into the existing https://iree.dev/guides/deployment-configurations/gpu-rocm/ page?

I think most of the content can apply to both. The differences would be in how to compile the runtime and which device to use.